lib/nip29/group_metadata.dart

```diff
@@ -75,4 +75,16 @@ class GroupMetadata extends GroupObject {
       open: open,
     );
   }
+
+  String? get displayName {
```

Comment:
🫨 function to get the display name was initially missing?

Reply:
This change was actually from before when I was working on the communities widget. Again, I just hadn't realized yet that this was inside a submodule.